src/vs/base/browser/ui/findinput/replaceInput.ts

```diff
@@ -370,7 +370,6 @@ export class ReplaceInput extends Widget {
 
 	public set width(newWidth: number) {
 		this.inputBox.paddingRight = this.cachedOptionsWidth;
-		this.inputBox.width = newWidth;
```

Comment:
Here I remove this line because `set paddingRight(paddingRight: number)` already calculates the input width with the right padding. Setting the width again is redundant.